src/main/java/org/jenkinsci/plugins/workflow/cps/CpsWhitelist.java

```diff
@@ -103,6 +104,9 @@ public boolean permitsStaticMethod(Method method, Object[] args) {
             String driverTo = "org.codehaus.groovy.runtime.";
             if (cn.startsWith(driverFrom)) {
                 try {
+                    if (Modifier.isPrivate(method.getModifiers())) {
+                        return true;
```

Comment:
Yes something along those lines.

Basically, when we are inside one of the transformed methods, it is senseless to be using the sandbox invoker. *But* need to verify that when we invoke a `Closure` it is still using the original invoker.